ios/FluentUI/Bottom Sheet/BottomSheetController.swift

```diff
@@ -23,8 +23,19 @@ public class BottomSheetController: UIViewController {
     /// By default the root view of `contentViewController` will be sized automatically to fill the available area,
     /// respecting the provided `preferredExpandedHeightFraction` multiplier.
     /// Alternatively, the content can size itself by setting `respectsPreferredContentSize` to true and providing a `preferredContentSize`.
-    @objc public init(with contentViewController: UIViewController) {
+    @objc public init(contentViewController: UIViewController) {
         self.contentViewController = contentViewController
+        self.contentView = contentViewController.view
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    /// Initializes the bottom sheet controller.
+    /// - Parameter contentView: The view that's placed inside the bottom sheet.
+    ///
+    /// TODO: Add view-specific sizing info
```

Comment:
is this related to how resize handle animates weirdly when rotating the device while expanded?

Reply:
Nope, but that is a thing I will look at as well.
This is because I added a way to just provide a content view to the bottom sheet rather than a content view controller. Views don't have the preferredContentSize prop, so I need to modify the API to reconcile that. It turns out we'll be adding multiple expansion states, so sizing will get more complicated and I'll do the API changes together with that because it's related.